scripts/fast-reboot-dump.py

```diff
@@ -147,6 +147,108 @@ def get_fdb(db, vlan_name, vlan_id, bridge_id_2_iface):
 
     return fdb_entries, available_macs, map_mac_ip
 
+def generate_fdb_entries_2(filename):
```

Comment:
Why do we have this change in fast-reboot?. @tahmed-dev for viz. fast-reboot enhancement is not mentioned in description. May require additional testing with some recent changes in this area

Reply:
is this a replacement of the generate_dfb_entries? if yes, what is the time saving here when using this version?

Reply:
Here, written and used optimized version (generate_fdb_entries_2) of generate_fdb_entries 
1) Modifed the functionality to work based on available mac addresses rather than available vlans 
2) used bulk (pipe) asic-db apis to get vlan and fdb from ASIC db instead of getting the data for individual vlan/fdb entry 
3) stored vlan_oid to vlan_id and vice-versa mapping in local memory instead of getting the same everytime from ASIC-DB 
4) done away with get_fdb function to be able to modify the overall logic and reduce copying of objects and function calls.

measured specifically fast-reboot-dump.py script from FDB perspective and now it takes around 10 sec instead of 12 minutes with 4K empty vlans;
Also tested the modified script with 4k mac addresses in 4k vlans (1 mac per vlan) and it gets completed with in 10 sec.